airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -320,10 +320,11 @@ def __handle_failed_task(self, task_arn: str, reason: str):
                 )
             )
         else:
-            self.log.error(
+            self.task_context_logger.error(
```

Comment:
do we really need to send this message? if the task is failed, the user will see that it is failed

Reply:
is this something different from task retries?  something like executor submit tries?

Reply:
> do we really need to send this message? if the task is failed, the user will see that it is failed

Fair

Reply:
> is this something different from task retries? something like executor submit tries?

Yes, it is a retry mechanism if the ECS task fails itself (thus, not able to start the Airflow task). If the ECS task succeeds and then the Airflow tasks fails, the task retry comes in play